plugins/packs-integrations.js

```diff
@@ -242,16 +241,19 @@ function generateCustomData(packsDescription) {
   return generatedCustomData;
 }
 
-function generateRoutes(packDataMap) {
-  return Object.keys(packDataMap).map((packName) => {
+function generateRoutes(packsAllData) {
+  return packsAllData.map((pack) => {
+   const parentVersion = pack.versions.find((version) => {
+      return version.children.find((child) => child.title === pack.latestVersion);
+    });
     return {
-      path: `/integrations/packs/${packName}`,
+      path: `/integrations/packs?pack=${pack.name}&version=${pack.latestVersion}&parent=${parentVersion?.title || pack.latestVersion}`,
```

Comment:
`parent=${parentVersion?.title}`
If there isn't a `parentVersion` from my perspective this query param should not be added.